README.md

```diff
@@ -219,76 +126,8 @@ kubectl get pods -n kube-system
 
 If both `vgpu-device-plugin` and `vgpu-scheduler` pods are in the *Running* state, your installation is successful.
 
-</details>
-
-<details> <summary> Upgrade </summary>
-
-Upgrading HAMi to the latest version is a simple process, update the repository and restart the chart:
-
-```
-helm uninstall hami -n kube-system
-helm repo update
-helm install hami hami-charts/hami -n kube-system
-```
-
-> **WARNING:** *If you upgrade HAMi without clearing your submitted tasks, it may result in segmentation fault.*
-
-</details>
-
-<details> <summary> Uninstall </summary>
-
-```
-helm uninstall hami -n kube-system
-```
-
-> **NOTICE:** *Uninstallation won't kill running tasks.*
-
-</details>
-
-### Submit Task
-
-<details> <summary> Task example </summary>
-
-Containers can now request NVIDIA vGPUs using the `nvidia.com/gpu`` resource type.
-
-```
-apiVersion: v1
```

Comment:
Would it be better if the quick start demo is retained? After the current changes, there is only the installation and introduction of hami, but no first example.

This yaml retention may give users a good first impression of hami, which is so easy to use.

Reply:
I add a link to examples here for user to verify their installation.